sdk/healthinsights/azure-healthinsights-cancerprofiling/README.md

```diff
@@ -0,0 +1,263 @@
+
+|SDK version|Supported API version of service |
+|-------------|---------------|
+|1.0.0b1 | 2023-03-01-preview|
+
+
+### Authenticate the client
+
+#### Get the endpoint
+
+You can find the endpoint for your Health Insights service resource using the [Azure Portal][azure_portal] or [Azure CLI][azure_cli]
+
+
+```bash
+# Get the endpoint for the Health Insights service resource
+az cognitiveservices account show --name "resource-name" --resource-group "resource-group-name" --query "properties.endpoint"
+```
+
+#### Get the API Key
+
+You can get the **API Key** from the Health Insights service resource in the Azure Portal.
+Alternatively, you can use **Azure CLI** snippet below to get the API key of your resource.
+
+```PowerShell
+az cognitiveservices account keys list --resource-group <your-resource-group-name> --name <your-resource-name>
+```
+
+#### Create a CancerProfilingClient with an API Key Credential
+
+Once you have the value for the API key, you can pass it as a string into an instance of **AzureKeyCredential**. Use the key as the credential parameter to authenticate the client:
+
+```python
+from azure.core.credentials import AzureKeyCredential
+from azure.healthinsights.cancerprofiling import CancerProfilingClient
+
+credential = AzureKeyCredential("<api_key>")
+client = CancerProfilingClient(endpoint="https://<resource-name>.cognitiveservices.azure.com/", credential=credential)
+```
+
+### Long-Running Operations
+
+Long-running operations are operations which consist of an initial request sent to the service to start an operation,
+followed by polling the service at intervals to determine whether the operation has completed or failed, and if it has
+succeeded, to get the result.
+
+Methods that support healthcare analysis, custom text analysis, or multiple analyses are modeled as long-running operations.
+The client exposes a `begin_<method-name>` method that returns a poller object. Callers should wait
+for the operation to complete by calling `result()` on the poller object returned from the `begin_<method-name>` method.
+Sample code snippets are provided to illustrate using long-running operations [below](#examples "Examples").
+
+## Key concepts
+
+The Cancer Profiling model allows you to infer cancer attributes such as tumor site, histology, clinical stage TNM categories and pathologic stage TNM categories from unstructured clinical documents.
+
+## Examples
+
+The following section (#cancer-profiling) provides code snippets that demonstrate usage of the CancerProfilingClient
+<!-- - See: [Cancer Profiling Samples][samples_location]-->
+
+### Cancer Profiling
+
+Infer key cancer attributes such as tumor site, histology, clinical stage TNM categories and pathologic stage TNM categories from a patient's unstructured clinical documents.
+
+```python
+from azure.core.credentials import AzureKeyCredential
+from azure.healthinsights.cancerprofiling.models import *
```

Comment:
we don't like to recommend `import *`, it's not pythonic. Instead, you can combine this with the following line and write
`from azure.healthinsights.cancerprofiling import CancerProfilingClient, models`. Then, for each model / enum, you would use it as `models.PatientInfo`

Reply:
Thanks for that.
Fixed